cmd/gazelle/update-repos.go

```diff
@@ -207,7 +208,7 @@ func updateRepos(args []string) (err error) {
 		macroPath = filepath.Join(c.RepoRoot, filepath.Clean(uc.macroFileName))
 	}
 	for f := range genForFiles {
-		if macroPath == "" && filepath.Base(f.Path) == "WORKSPACE" ||
+		if macroPath == "" && strings.Contains(filepath.Base(f.Path), "WORKSPACE") ||
```

Comment:
`strings.Contains` isn't right. It shouldn't match `aaaWORKSPACEbb` for example.

Perhaps `wspace` should provide a function `func IsWORKSPACE(path) bool` that can be used here.